react-app/src/components/ProposalScreen/ProposalScreen.tsx

```diff
@@ -76,6 +92,14 @@ const ProposalScreen: React.FC = () => {
     fetch(offset);
   }, [fetch, offset]);
 
+  useEffect(() => {
+    const page = offset / PROPOSAL_LIST_PAGE_SIZE + 1;
+    setSearchParams({
```

Comment:
@hochiw will this populate browser history tho? what if user press back btn?

Reply:
Added `replace: true` to prevent param reverting

Reply:
oh... i mean populating history is what we want? so user can press forward / backward as expected

Reply:
oh ok